### PR TITLE
[IMP] website: redesign s_table_of_content snippet

### PR DESCRIPTION
**Pull request by chgo-odoo**
task-3665306

| Master | This PR |
|--------|--------|
| image | image | 

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by chgo-odoo** [comment]
| heading 4 | heading 5 |
|--------|--------|
| image | image |

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Thanks @chgo-odoo  👍 
I do agree with you `h5` are better indeed. 
Thanks!

File: addons/website/views/snippets/s_table_of_content.xml [hunk 1]
```diff
@@ -16,36 +16,43 @@
                 <div class="col-lg-9 s_table_of_content_main oe_structure oe_empty" data-name="Content">
                     <section class="s_text_block pt0 pb16" data-snippet="s_text_block" data-name="Text">
                         <div class="container s_allow_columns">
-                            <h2 id="table_of_content_heading_1_1" data-anchor="true">Intuitive system</h2>
+                            <h2 id="table_of_content_heading_1_1" class="h3" data-anchor="true">Intuitive system</h2>
+                            <div class="s_hr pt24 pb24">
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply addons/website/views/snippets/s_table_of_content.xml:20]: Make sense, indeed ! It's weird to have an even spacing on both sides

**Review by stefanorigano — APPROVED** [review]
Thanks @chgo-odoo 👍

**Comment by chgo-odoo** [comment]
@stefanorigano had to change some changes here : 

- Restore the second section to showcase the sticky feature
- Slightly increased the padding between the section

**Comment by stefanorigano** [comment]
Re-approved :)
Thanks for the improvements and the heads-up 👍

**Review by xO-Tx — CHANGES_REQUESTED** [review]

**Comment by chgo-odoo** [comment]
Hello @xO-Tx, thanks for having a look at this PR, changes have been applied !

Thanks !

**Comment by detrouxdev** [comment]
@robodoo delegate=xO-Tx

**Review by xO-Tx — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**